src/components/AccessForm.vue

```diff
@@ -1,55 +1,95 @@
 <template>
-  <el-form label-width="auto" :model="form">
-    <el-form-item label="Key">
+  <el-form label-width="auto" :model="form" :rules="rules" ref="accessForm">
+    <el-form-item label="Key" prop="access_key">
       <el-input
         v-model="form.access_key"
         placeholder="Data access key"
-        required
+        type="password"
+        autocomplete="off"
       ></el-input>
     </el-form-item>
-    <el-form-item label="UUID">
+    <el-form-item label="UUID" prop="uuid">
       <el-input
         v-model="form.uuid"
         placeholder="Data UUID / Identifier"
-        required
+        autocomplete="off"
       ></el-input>
     </el-form-item>
-    <el-form-item label="Hash">
+    <el-form-item label="Hash" prop="hash">
       <el-input
         v-model="form.hash"
         placeholder="SHA-256 of the binary to authorise"
-        required
+        autocomplete="off"
       ></el-input>
     </el-form-item>
     <el-form-item label="Number of uses">
       <el-input-number
-        v-model="form.nofu"
+        v-model.number="form.nofu"
         :min="1"
         :max="100"
       ></el-input-number>
     </el-form-item>
     <el-form-item style="margin-top: 15px">
-      <el-button type="primary" @click="onSubmit">Create</el-button>
+      <el-button type="primary" @click="onSubmit('accessForm')"
+        >Create</el-button
+      >
     </el-form-item>
   </el-form>
 </template>
 
 <script lang="ts">
 import { defineComponent } from "vue";
+import { encryptBlob } from "@/utils/cryptography";
+import elForm from "element-plus/lib/el-form";
+
 export default defineComponent({
   data() {
     return {
       form: {
-        access_key: "",
-        uuid: "",
-        hash: "",
+        access_key: null,
+        uuid: null,
+        hash: null,
         nofu: 1
+      },
+      rules: {
+        access_key: [
+          {
+            required: true,
+            message: "Access key is required",
+            trigger: "change"
+          }
+        ],
+        uuid: [
+          {
+            required: true,
+            message: "UUID is required",
+            trigger: "change"
+          }
+        ],
+        hash: [
+          {
+            required: true,
+            message: "Hash is required",
+            trigger: "change"
+          }
+        ]
       }
     };
   },
   methods: {
-    onSubmit() {
-      console.log("AccessForm submit:", this.form);
+    onSubmit(formName: string) {
+      (this.$refs[formName] as typeof elForm).validate((valid: boolean) => {
+        if (valid) {
+          const encodedForm = new TextEncoder().encode(
+            JSON.stringify(this.form)
+          );
+          const enclavePublicKey = this.$store.getters.enclavePublicKey;
+          encryptBlob(encodedForm, enclavePublicKey);
```

Comment:
Missing await here: we should probably at least log the result for now, and await to avoid silencing any unexpected error?